### PR TITLE
DOCSP-6636: Kafka Docker Example

### PR DESCRIPTION
**Pull request by ccho-mongodb**
JIRA: https://jira.mongodb.org/browse/DOCSP-6636

Staging: https://docs-mongodbcom-staging.corp.mongodb.com/ecosystem/docsworker/DOCSP-6636-kafka-docker-example/connectors/kafka-docker-example.html

### CODE REVIEW

**Comment by terakilobyte** [comment]
DOCSP-6636

File: source/connectors/kafka-docker-example.txt [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+* The **mongo-source** connector produces change events for the
+  "test.pageviews" collection and publishes them to the
+  "mongo.test.pageviews" collection.
+
+Requirements
+------------
+
+Linux/Unix-based OS
+~~~~~~~~~~~~~~~~~~~
+* Docker 18.09 or later
```
Comment (terakilobyte) [inline source/connectors/kafka-docker-example.txt]: Can we link to installation instructions? The ones on the docker website worked fine for me on 18.04 https://docs.docker.com/compose/install/

File: source/connectors/kafka-docker-example.txt [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+How to Run the Example
+----------------------
+
+Clone the `mongo-kafka <https://github.com/mongodb/mongo-kafka>`_ repository
+from GitHub:
+
+.. code-block:: shell
+
+   git clone git@github.com:mongodb/mongo-kafka.git
```
[the remark this answers is not in the document]
Comment (ccho-mongodb) [reply source/connectors/kafka-docker-example.txt]: Sounds good, will fix.

File: source/connectors/kafka-docker-example.txt (hunk 1 above)
Comment (ccho-mongodb) [reply source/connectors/kafka-docker-example.txt]: Ok sure. Did you have an issue with the version? The original docs mentioned 18.09, so I just used that. If you want to change it to 18.04 and you have confirmed that it works, that's fine with me.

**Comment by ccho-mongodb** [comment]
Ready for re-review @terakilobyte

**Comment by terakilobyte** [comment]
lgtm

**Pull request merged**